crates/ln-dlc-node/src/tests/just_in_time_channel.rs

```diff
@@ -26,9 +26,13 @@ async fn just_in_time_channel() {
     payer.fund(Amount::from_sat(100_000)).await.unwrap();
     coordinator.fund(Amount::from_sat(100_000)).await.unwrap();
 
-    let payer_outbound_liquidity_sat = 50_000;
-    let payer_coordinator_channel_details = payer
-        .open_channel(coordinator.info, payer_outbound_liquidity_sat, 0)
+    let payer_outbound_liquidity_sat = 25_000;
+    let payer_coordinator_channel_details = coordinator
+        .open_channel(
+            &payer,
+            payer_outbound_liquidity_sat,
+            payer_outbound_liquidity_sat,
+        )
         .await
         .unwrap();
```

Comment:
:wrench: I think it's clearer if we only fund one side (the payer). Although it might not be possible given that we let the coordinator initiate the channel creation. At the very least, we should not fund the coordinator with the `payer_outbound_liquidity_sat` variable.

Reply:
https://github.com/get10101/10101/pull/169/commits/dc8c5e2b43923035927af3e2710f5e9e821052c9